docs/workflows/sync.rst

```diff
@@ -0,0 +1,106 @@
+Synchronize a Repository
+========================
+
+Users can populate their repositories with content from an external source like PyPI by syncing
+their repository.
+
+Create a Repository
+-------------------
+
+Start by creating a new repository named "foo"::
+
+    $ http POST http://localhost:8000/pulp/api/v3/repositories/ name=foo
+
+Response::
+
+    {
+        "_href": "http://localhost:8000/pulp/api/v3/repositories/e81221c3-9c7a-4681-a435-aa74020753f2/",
+        ...
+    }
+
+If you want to copy/paste your way through the guide, create an environment variable for the repository URI.::
+
+    $ export REPO_HREF=$(http :8000/pulp/api/v3/repositories/ | jq -r '.results[] | select(.name == "foo") | ._href')
```

Comment:
for the sake of consistency, why not use $PULP_BASE (or whatever the variable name will end up being).